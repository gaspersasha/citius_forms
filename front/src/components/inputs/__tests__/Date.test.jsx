import React from 'react';
import { InputDate } from '..';

const defaultProps = {
  onChange: jest.fn(),
};

describe('<InputDate />', () => {
  let wrapper;

  const getWrapper = (props) =>
    shallow(<InputDate {...defaultProps} {...props} />);

  it('renders without errors', () => {
    wrapper = getWrapper();
    expect(wrapper).toMatchSnapshot();
  });
});
