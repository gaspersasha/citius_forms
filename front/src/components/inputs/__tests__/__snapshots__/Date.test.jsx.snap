// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputDate /> renders without errors 1`] = `
<Label
  customClasses=""
  name="datepicker"
  required={false}
  status="default"
  title="Date of birth"
  withSub={false}
>
  <t
    className=""
    guide={true}
    mask={[Function]}
    name="datepicker"
    onChange={[MockFunction]}
    pipe={[Function]}
    placeholder="DD/MM/YYYY"
    render={[Function]}
  />
</Label>
`;
